tests/docker-compose.yml

```diff
@@ -1,67 +1,70 @@
-version: '3.2'
-
-# Only one of these will be up at a time for now.
-# Concurrency will be on the tests folder we have.
-
 services:
   modsec2-apache:
     container_name: modsec2-apache
     image: owasp/modsecurity-crs:apache
+    # NOTE: The user used to run the container process is explicitly set to
+    # 'root'. This fixes issues with permissions on the logging directories used
+    # as bind mounts. This is done as *a convenience for running the CRS testing
+    # setup only* and *should not be done in general!*
+    user: root
```

Comment:
Why is this showing as a new change? I had added this weeks ago...

Reply:
In the v3/dev branch?

Reply:
Oh, I didn't see that.